src/components/SkipNavigation.js

```diff
@@ -0,0 +1,35 @@
+import React from "react"
+import styled from "styled-components"
+import { useTranslation } from 'react-i18next'
+
+const Anchor = styled.a`
+& {
+  position: absolute;
+  left: -999px;
+  top: -999px;
+  background-color: white;
+  border: 1px solid;
+  padding: 0.5rem;
+}
+
+&:focus {
+  left: 2rem;
+  top: 0.5rem;
+}
+`
+
+const SkipNavigation = () => {
```

Comment:
@voluntadpear why this component?

Reply:
Quesion: What is it doing?

Reply:
![image](https://user-images.githubusercontent.com/10501948/77939069-963c3880-7284-11ea-9aaf-7678c7746c81.png)

It adds a link as the first focusable element on the page to aid keyboard users to skip the navigation links if they wish for.

See https://webaim.org/techniques/skipnav/